src/models/VatRate.ts

```diff
@@ -1 +1,3 @@
 export type VatRate = 0 | 5 | 8 | 23;
+
+export const vatRates: VatRate[] = [0, 5, 8, 23];
```

Comment:
This should be moved to other directory, because it's not a model :upside_down_face: I'm not sure, maybe `src/calculation` or a new directory (for example `src/constants`).

Reply:
I don't see that change yet :thinking:

Reply:
Now I see :smile: